src/main/features/offer/views/your-offer.njk

```diff
@@ -10,16 +10,19 @@
       <form novalidate method="post">
         {{ csrfProtection(csrf) }}
 
-        <p>{{ t('Make it clear what you will do and by when. For example: "I will pay you £240 on 29 April 2018 by bank transfer".') }}</p>
+        <p>{{ t('Make it clear what you will do and by when. For example: "I will pay you £240 on {{ nextMonth }} by bank transfer".',
+            {nextMonth: 'now' | addDays(30) | inputDate}) }}</p>
 
-        <p>{{ t('If you’re also asking {{ claimantName }} to do something, say clearly what it is and by when it should be done. For example: "You will fix the roof by 28 April 2018."', { claimantName: claim.claimData.claimant.name }) }}</p>
+        <p>{{ t('If you’re also asking {{ claimantName }} to do something, say clearly what it is and by when it should be done. For example: "You will fix the roof by {{ twoWeeksLater }}."',
+            { claimantName: claim.claimData.claimant.name, twoWeeksLater: 'now' | addDays(44) | inputDate }) }}</p>
```

Comment:
Adding 44 days does not make it two weeks later

Reply:
I meant two weeks after the 30 days from the previous calculation

Reply:
why not just 14 days?